codec-http/src/main/java/io/netty/handler/codec/http/multipart/HttpPostMultipartRequestDecoder.java

```diff
@@ -805,6 +796,28 @@ private InterfaceHttpData findMultipartDisposition() {
         }
     }
 
+    private static final String FILENAME_ENCODED = HttpHeaderValues.FILENAME.toString() + '*';
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "DOUBLE_SINGLE_QUOTE" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1068)